_pages/push-notifications.md

```diff
@@ -15,19 +15,26 @@ sidenav:
     href: "#verify-signature"
 ---
 
-# Setup URL
+# How it works
 
-Set up your push_notification_url for your app in the partner dashboard. For production, this will be an application setting that you supply to login.gov.
+When a user associated with your application deletes their account, login.gov will make a post request to the endpoint URL that you specified in the previous step. The body of the request will be an empty JSON document. The headers of the request will contain the relevant information.
 
-# Receive request
+# Configuration
 
-When a user associated with your application deletes their account, login.gov will make a post request to the endpoint URL that you specified in the previous step. The body of the request will be an empty JSON document. The headers of the request will contain the relevant information.
+## Set up the url
+Set up your push_notification_url for your app in the partner dashboard. For production, this will be a configuration you supply to login.gov.
+
+## Fetch public key
```

Comment:
section should be in usage since it should be called dynamically (not just once) as part of the flow